developers/README.md

```diff
@@ -2,28 +2,23 @@
 
 ## Table of Contents
 - [Development Environment](#development-environment)
-  - [Local Environment](#local-environment)
-  - [Virtual Machine](#virtual-machine)
+  - [Virtual Machines](#virtual-machines)
   - [Docker](#docker)
 - [Contributing](#contributing)
 - [IRC](#irc)
 
 ## Development Environment
-Before hacking on BonnyCI, we'll need to set up a development environment. There are three basic ways to do so:
-  1. [On our local machine](#local-environment)
-  2. [In a virtual machine](#virtual-machine)
-  3. [In a Docker container](#docker)
+Before hacking on BonnyCI, we'll need to set up a development environment. There are two basic ways to do so:
+  2. [In virtual machines](#virtual-machines)
+  3. [In Docker containers](#docker)
 
-### Local Environment
+### Virtual Machines
 Supported development platforms:
   - [Ubuntu](dev-environment/ubuntu.md)
```

Comment:
That I understand. What I'm saying is, `ubuntu.md` and `macOS.md` are for getting local or remote host machines ready to start BonnyCI in a virtualized or containerized environment, whereas the `Virtual Machines` and `Docker` subsections would be used for instructions on how to actually start BonnyCI in the relevant way.

So, `Development Environment` has the instructions to install prerequisites (like vagrant), whereas `Virtual Machines` has instructions to start the environment (git clone hoist, navigate to the dir and `vagrant up`).